man/repart.d.xml

```diff
@@ -669,15 +669,23 @@
         all partition types that support it, except if the partition is marked read-only (and thus
         effectively, defaults to off for Verity partitions).</para></listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><varname>SplitName=</varname></term>
```

Comment:
In systemd/mkosi#1181 the complete filename of split artifacts can be specified independent of the main artifact. What is the reason for `SplitName=` to only configure the suffix?

Admittedly most of the time you want all the files to share the same prefix, but specifying the complete name is more flexible and arguably easier to implement.

Reply:
Doing it like this allows you to change the prefix without having to change the configuration files. The prefix can be configured by changing the name of the loopback file you pass to the repart command line. The suffix is configured with SplitName=